tasks/arrayToObject.js

```diff
@@ -0,0 +1,21 @@
+// get the old array, store it in json variable
+const json = require('../data/oldCardsArray.json');
+const fs = require('fs');
+
+// new object to write
+const newJson = {};
+
+// iterate through the array
+for (let i = 0; i < json.length; i += 1) {
```

Comment:
`sort()` the array (alphabetically in ascending order) before you iterate through it.